examples/multiple-prefixes/main.tf

```diff
@@ -0,0 +1,40 @@
+module "s3logging-bucket" {
+  source = "../.."
+
+  bucket_suffix = "system"
+
+  lifecycle_rules = [{
+    id      = "cloudtrail-expire"
+    enabled = true
+    prefix  = "1111111111111-us-east-1-cloudtrail"
+
+    # keep all cloudtrail logs for 5 years
+    expiration                    = 1825
+    noncurrent_version_expiration = 1825
+    },
```

Comment:
In CloudWatch it would be, but should be pretty minimal in S3. Plus, it _is_ a requirement for most compliance scenarios. But yeah, it's kinda useless to have 5 years of gzip'd fragmented JSON..